python_testcontainers/infrahub_testcontainers/docker-compose.test.yml

```diff
@@ -28,6 +28,22 @@ services:
       timeout: 5s
       retries: 3
 
+  loadbalancer:
```

Comment:
not a blocker but I think it would be good to name it `infrahub-server-lb` or similar to be more explicit about is its role. I think most people won't be able to understand right away what this component is about.

Reply:
I initially went with `infrahub-server-lb` so I'm ok with that :+1: